src/components/PrioritySelector/PrioritySelector.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from 'react';
+import Close from '@material-design-icons/svg/round/close.svg';
+import KeyboardDoubleArrowUp from '@material-design-icons/svg/round/keyboard_double_arrow_up.svg';
+import KeyboardArrowUp from '@material-design-icons/svg/round/keyboard_arrow_up.svg';
+import KeyboardArrowDown from '@material-design-icons/svg/round/keyboard_arrow_down.svg';
+import { DropdownMenuItem, Root, Portal } from '@radix-ui/react-dropdown-menu';
+import { DropdownTrigger } from '../Dropdown/DropdownTrigger';
+import { DropdownContent } from '../Dropdown/DropdownContent';
+import { bem } from '../../utils/bem';
+import { Text } from '../Text';
+import { IconButton } from '../Buttons';
+import itemStyles from '../Dropdown/Items/Item.scss';
+import styles from './PrioritySelector.scss';
+
+const iconMap = {
+    mandatory: KeyboardDoubleArrowUp,
+    important: KeyboardArrowUp,
+    optional: KeyboardArrowDown,
+    exclude: Close,
+};
+
+export enum Priority {
```

Comment:
I'd avoid using the enum type. It would only cause problems at the application level because we would need to use the same enum type even if the values are the same. Instead, let's use a union type. There are many articles about enums: https://www.totaltypescript.com/why-i-dont-like-typescript-enums

Reply:
it's not only one article :) Union types also covers that cases:"so avoid typos and similar errors."

I don't see many UI libraries that exported enum. It's easy type string state in the props that export one more TS type and use it:

```
import { Component } from 'library';
<Component type="mandatory" />
```
vs 
```
import { Component, ComponentTypes } from 'library';
<Component type={ComponentTypes.Mandatory} />
```
All samples are type safe.

Reply:
ok, let's go with the `string` solution. Janneke already implemented it anyway :-)